sdk/log/record.go

```diff
@@ -233,3 +234,39 @@ func (r *Record) Clone() Record {
 	res.back = slices.Clone(r.back)
 	return res
 }
+
+// MarshalJSON marshals the Record to JSON.
+func (r *Record) MarshalJSON() ([]byte, error) {
+	return json.Marshal(recordJSON{
```

Comment:
This is missing tests.

Reply:
It seems the cross-package test does not count as covered in Codecov